docs/gui.rst

```diff
@@ -74,39 +75,40 @@ Paint a window
 
     The data type of ``img`` must be one of:
 
-    * float32, clamped into [0, 1]
+    * ``float32``, range ``[0, 1]``
 
-    * float64, clamped into [0, 1]
+    * ``float64``, range ``[0, 1]``
 
-    * uint8, range [0, 255]
+    * ``uint8``, range ``[0, 255]``
 
-    * uint16, range [0, 65535]
+    * ``uint16``, range ``[0, 65535]``
 
-    * uint32, range [0, UINT_MAX]
+    * ``uint32``, range ``[0, 4294967295]``
 
```

Comment:
Let's say something about the #1005 - if a float value is not in range `[0, 1]`, it will be clip into that range.
```suggestion
    .. note ::
    
        If a float value is not in the range ``[0, 1]``, it will be clipped into that range, e.g.: ``1.3 -> 1``, ``-0.2 -> 0``, ``0.5 -> 0.5``.
        
```